src/firecracker/src/main.rs

```diff
@@ -150,11 +168,9 @@ fn main() {
 
     // It's safe to unwrap here because clap's been provided with a default value,
     // and allowed values are guaranteed to parse to u32.
-    let seccomp_level = cmd_arguments
-        .value_of("seccomp-level")
-        .unwrap()
-        .parse::<u32>()
-        .unwrap();
+    let seccomp_level = cmd_arguments.value_of("seccomp-level").unwrap();
+    let seccomp_filter =
+        get_seccomp_filter(seccomp_level).expect("Could not create seccomp filter");
```

Comment:
This `expect` panics, so you can remove the `panic!` on line 55 (`fn get_seccomp_filter`) and propagate a `Result`.